src/lib/pmd/PmdCatalogWrapper.ts

```diff
@@ -151,32 +149,20 @@ export class PmdCatalogWrapper extends PmdSupport {
 		return path.join(PMD_LIB, "pmd-" + language + "-" + PMD_VERSION + ".jar");
 	}
 
-	/**
-	 * Accepts a child process created by child_process.spawn(), and a Promise's resolve and reject function.
-	 * Resolves/rejects the Promise once the child process finishes.
-	 * @param cp
-	 * @param res
-	 * @param rej
-	 */
-	protected monitorChildProcess(cp: ChildProcessWithoutNullStreams, res: (string) => void, rej: (string) => void): void {
```

Comment:
The output processor functionality was moved upwards into `CommandLineSupport`, making this method unnecessary.